actioncable/lib/action_cable.rb

```diff
@@ -32,7 +32,8 @@ module ActionCable
     message_types: {
       welcome: 'welcome'.freeze,
       ping: 'ping'.freeze,
-      confirmation: 'confirm_subscription'.freeze,
+      confirm_subscribe: 'confirm_subscribe'.freeze,
+      confirm_unsubscribe: 'confirm_unsubscribe'.freeze,
```

Comment:
Could also go with `unsubscribe_confirmation` and keep the existing `confirmation`.